draft-ietf-ice-rfc5245bis.xml

```diff
@@ -678,9 +678,9 @@ responding agent, the peer is the initiating agent. </t>
 transport protocol (such as UDP or TCP) port.</t>
 
 <t hangText="Media, Media Stream, Media Session:"> When ICE is used to
-setup multimedia sessions, the media is usually transported over RTP,
+setup communication sessions, the media is usually transported over RTP,
```

Comment:
For this particular instance, it should stay "multimedia", I think.

Reply:
Ok.